brambling/models.py

```diff
@@ -1459,6 +1459,9 @@ class Attendee(AbstractNamedModel):
 
     custom_data = GenericRelation('CustomFormEntry', content_type_field='related_ct', object_id_field='related_id')
 
+    # Admin-only data
```

Comment:
I'm gonna change this to say "Organizer-only data". We're admins. Our customers are organizers.